mobile/app/src/main/java/hu/bme/caffshare/ui/admin/CaffListPresenter.kt

```diff
@@ -0,0 +1,55 @@
+package hu.bme.caffshare.ui.admin
+
+import co.zsmb.rainbowcake.withIOContext
+import hu.bme.caffshare.ui.admin.model.User
+import hu.bme.caffshare.ui.cafflist.model.CaffFile
+import javax.inject.Inject
+
+class UserListPresenter @Inject constructor() {
+
+    suspend fun getUsers(): List<User> = withIOContext {
```

Comment:
Én úgy csinátam eddig, hogy a visszatérési érték _nullable_, és akkor null érték esetén be lehet állítani a hiba képernyőt

Reply:
Átírtam a javaslatnak megfelelően.